modules/cli/src/main/java/sap/prd/cmintegration/cli/Commands.java

```diff
@@ -117,7 +117,7 @@ static BackendType getBackendType(CommandLine commandLine) {
             try {
                 return BackendType.valueOf(commandLine.getOptionValue(CMOptions.BACKEND_TYPE.getOpt()));
             } catch(IllegalArgumentException e) {
-                throw new RuntimeException("Cannot retrieve backend type.", e);
+                throw new RuntimeException("Can not retrieve backend type.", e);
```

Comment:
_These two spellings [cannot/can not] are largely interchangeable, but by far the most common is “cannot” and you should probably use it except when you want to be emphatic: “No, you can not wash the dog in the Maytag.”_

Reply:
fixed, thanks